src/main/java/com/toptrumps/core/DeckParser.java

```diff
@@ -1,65 +1,84 @@
 package com.toptrumps.core;
 
 
+import com.sun.xml.internal.bind.v2.TODO;
+
 import java.io.*;
 import java.util.ArrayList;
 
 
-// TODO: Remove this class
 public class DeckParser {
 
     private ArrayList<Card> cards;
+    private final String FILE_PATH = "src/main/resources/assets/StarCitizenDeck.txt";
+
+
+    /**
+     * Builds the deck for the game.
+     *
+     */
 
     public DeckParser() {
-        buildDeck(); // calls the build deck method
+        buildDeck();
     }
 
+    /**
+     * Gets the cards from the specified card location,
+     * and sets the "deck" in an ArrayList called cards.
+     * @return void
+     */
+
     private void buildDeck() {
-        cards = new ArrayList<>(); // creates a new array list
+        cards = new ArrayList<>();
+
 
-        // creates a new File called cardsFile at the location of the file to be read in.
-        File cardsFile = new File("src/main/resources/assets/StarCitizenDeck.txt");
+        File cardsFile = new File(FILE_PATH);
 
         try {
             FileReader fr = new FileReader(cardsFile);
             BufferedReader br = new BufferedReader(fr);
 
             try {
-                String line = br.readLine(); // reads the first line of the file (contains the headers).
-                String[] categories = line.split(" "); // splits the file based on the spaces into an array called categories.
```

Comment:
These comments were actually useful! Because it explains the parsing logic step-by-step. It would be ok to leave them here.